.github/workflows/deploy-docker.yml

```diff
@@ -36,18 +35,33 @@ jobs:
             -
                 name: Get branch name
                 id: branch-name
-
                 uses: tj-actions/branch-names@v6
 
 
             -
                 name: Set tag for non-default branch
-                if: steps.branch-name.outputs.is_default == 'false' && steps.branch-name.outputs.default_branch != ''
+                if: ${{ steps.branch-name.outputs.is_default == 'false' && steps.branch-name.outputs.default_branch != '' && github.actor != 'dependabot[bot]' }}
                 run: |
+                    echo "Run me: (${{ steps.branch-name.outputs.is_default == 'false' && steps.branch-name.outputs.default_branch != '' && github.actor != 'dependabot[bot]' }})"
+                    echo "Set tag for PR: ${{ github.event_name == 'pull_request' || github.actor == 'dependabot[bot]' && steps.branch-name.outputs.is_default == 'false' && steps.branch-name.outputs.default_branch != '' }} "
+                    echo "Actor = ${{ github.actor }}"
+                    echo "event name is ${{ github.event_name }}"
+                    echo "PR_NUMBER=$(echo $GITHUB_REF | awk 'BEGIN { FS = "/" } ; { print $3 }')"
+                    echo "PR NAME: ${GITHUB_REF##*/}"
+                    echo "PR number: ${{ github.event.pull_request.number }}"
                     echo "Branch name is ${{ steps.branch-name.outputs.ref_branch }}"
                     echo "Main name is ${{ steps.branch-name.outputs.default_branch }}"
                     echo "TAG=${{ steps.branch-name.outputs.ref_branch }}" >> $GITHUB_ENV
-
+            -
+                name: Set tag for PR
+                if: ${{ github.event_name == 'pull_request' || github.actor == 'dependabot[bot]' && steps.branch-name.outputs.is_default == 'false' && steps.branch-name.outputs.default_branch != '' }}
```

Comment:
Sets a valid version of github tag changing `102/merge` to `pr102`
This will also push the tag version to the VERSION tag